hashintel/hash
# Fix production Docker compose and images

## 🌟 What is the purpose of this PR?



This PR fixes problems with the production docker compose and the images used by it.

The CD pipeline should be fixed by this as well

## 🔗 Related links





- [Asana task](https://app.asana.com/0/1204315348455207/1204588404823948/f) _(internal)_

## 🚫 Blocked by





## 🔍 What does this change?




- Fixes API prod docker compose (using turbo graph and adding Python for the agent typings)
- Fixes Frontend prod docker compose (also using turbo graph and addign Python for the agent typings)
- Adjusts the docker-compose to properly make use of the graph image (changing the `command`)
- Adds the type fetcher to the docker compose

## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:












### 📜 Does this require a change to the docs?




The changes in this PR:






### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:



- [x] Does change the execution graph, but not because of the change. I have a suspicion that the graph was misrepresenting the entry that was changed for API codegen



## ⚠️ Known issues



## 🐾 Next steps



## 🛡 What tests cover this?



- Manual running

## ❓ How to test this?



1.  Checkout the branch
1.  Run `docker compose --file apps/hash-external-services/docker-compose.prod.yml --env-file .env --env-file .env.local up --build` from the root of the repo
1.  Confirm that the service eventually starts at http://localhost:3000

## 📹 Demo
